jkanban.js

```diff
@@ -516,6 +519,9 @@ var dragula = require('dragula');
     }
 
     function __onContextHandler(nodeItem, contextfn) {
+      if (typeof self.options.context !== 'function') {
```

Comment:
And then here we can check ownedHandlers as well like ´if (!self.ownedHandlers.includes('context')) { return }`